sage-ios/SAGE/SAGE/LoginHelper.swift

```diff
@@ -7,30 +7,148 @@
+        let retrievedUsername = KeychainWrapper.objectForKey(KeychainConstants.kVEmail)
         if let _ = retrievedUsername {
             return true
         }
         return false
     }
     
-    static func userIsVerified() -> Bool {
-        // this returns only true for now because it hasn't been connected to the backend
-        if let _ = KeychainWrapper.stringForKey(KeychainConstants.kVerified) {
-            return true
+    static func userIsVerified(completion: ((Bool) -> Void)) {
+        if let verified = KeychainWrapper.objectForKey(KeychainConstants.kVerified) {
+            let verifiedBool = verified as! Bool
+            if verifiedBool {
+                completion(true)
+            } else {
+                if let verified = User.currentUser?.verified {
+                    completion(verified)
+                } else {
+                    let email = KeychainWrapper.objectForKey(KeychainConstants.kVEmail) as? String
+                    let password = KeychainWrapper.objectForKey(KeychainConstants.kVPassword) as? String
+                    if (email != nil && password != nil) {
+                        LoginHelper.isValidLogin(email!, password: password!, completion: { (success) -> Void in
+                            if success && User.currentUser!.verified! {
+                                completion(true)
+                            } else {
+                                completion(false)
+                            }
+                        })
+                    } else {
+                        completion(false)
+                    }
+                }
+            }
+        } else {
+            completion(false)
+        }
+    }
+    
+    static func storeUserDataInKeychain(firstName: String?, lastName: String?, email: String?, password: String?, school: Int?, hours: Int?, verified: Bool?) {
+        if let firstName = firstName {
+            KeychainWrapper.setObject(firstName, forKey: KeychainConstants.kVFirstName)
+        }
+        if let lastName = lastName {
+            KeychainWrapper.setObject(lastName, forKey: KeychainConstants.kVLastName)
+        }
+        if let email = email {
+            KeychainWrapper.setObject(email, forKey: KeychainConstants.kVEmail)
+        }
+        if let password = password {
+            KeychainWrapper.setObject(password, forKey: KeychainConstants.kVPassword)
+        }
+        if let hours = hours {
+            KeychainWrapper.setObject(hours, forKey: KeychainConstants.kVHours)
+        }
+        if let verified = verified {
+            KeychainWrapper.setObject(verified, forKey: KeychainConstants.kVerified)
+        }
+    }
+    
+    static func createUser(firstName: String, lastName: String, email: String, password: String, school: Int, hours: Int, role: Int, photoData: String, completion: ((Bool) -> Void)) {
+        
+        let operationManager = AFHTTPRequestOperationManager()
```

Comment:
use the BaseOperation.manager() method, and maybe put this in some UserOperation class? i guess keeping it in this login helper is also ok though